twitchpopups.htm

```diff
@@ -16,6 +16,20 @@
         // The emoji that surrounds the hotseat messages.
         const hotseatEmoji = '🔥';
 
+        // Start displaying timed messages as soon as the source is added? '!timed on', '!timed true' or '!timed start' will start them if not.
+        const timedMessagesAtStart = false;
+
+        // The timed messages to rotate through displaying
+        const timedMessages = ['Timed message 1',

```

Comment:
Might be nice for mods to be able to view, remove and push new options to this list rather than relying on hardcoded values

Reply:
Yeah I think it could be extended in different ways - not sure if it's worth waiting to see if Limmy wants it in first?